definitions/features/hammer.rb

```diff
@@ -80,6 +80,15 @@ def run(args)
     execute("#{command_base} #{args}")
   end
 
+  def admin_username
+    return nil unless feature(:installer)
```

Comment:
shouldn't we return 'admin' instead of nil?

Reply:
makes sense, thanks @mbacovsky 
will update pr.

Reply:
updated pr.